core/src/main/java/nl/nn/adapterframework/doc/IbisDocPipe.java

```diff
@@ -312,9 +312,15 @@ private static void addExcludeFilter(ClassPathBeanDefinitionScanner scanner, Str
 	private static Class<?> getClass(String className) {
 		try {
 			return Class.forName(className);
-		} catch (Throwable t) {
-			errors.put(className, t.getClass() + ": " + t.getMessage());
+		} catch (NoClassDefFoundError e) {
+			// This exception happens when you have the proprietary sub-projects of the Frank!Framework.
+			// These sub-projects have classes that depend on third-party classes. If such a third-party
+			// class is not found, then this exception handler is entered. We ignore the error because
+			// we do have the class in the proprietary FF! subproject.
 			return null;
```

Comment:
Ik denk dat het wel goed is om ergens te kunnen zien dat deze Exceptie is opgetreden, bijvoorbeeld in de logging, door hier iets op te nemen als:
log.warn("ignoring NoClassDefFoundError, assuming it is about a third party class", e);
Als het namelijk een keer toch over een framework class gaat, dan zoek je je anders een aap.